### ISSUE TITLE
How to increase execution speed - multiple cores, remote server?

### ISSUE THREAD

**Issue by KapishAggarwal** [Tudat/tudat#609]
Hi

I have the following queries with regards to increasing speed during runtime - 

1. Is there a way to increase the propagation speed by changing settings in QtCreator? Can multiple cores/parallel computations be somehow used when running an application? (I assume using "-jX" in the tools argument is only useful during compilation and not for runtime.)
2. Is it beneficial (in terms of speed) to use the TU Delft remote server? (Given that my (Windows) laptop has 16GB RAM and 12 logical cores, which only seems to be utilized 50% during the propagations.) Can multiple cores be used on the linux server? Or maybe multiple applications be run simultaneously to reduce total runtime? 

Thanks!

Kapish

**Comment by DominicDirkx** [comment]
Dear Kapish,

Please explain the problem you are trying to solve in some more detail, and also add a link to the repository with the code you are currently using. Based on that, we can give a more informed answer to your question,

Best,

Dominic

**Comment by KapishAggarwal** [comment]
Dear Dominic

I'll give you an example of what I have done so far.

I propagated 5088 trajectories around Mars using symplectic integrator. The propagation duration was 200 years and step size was 0.5 Julian days. Each propagation took about 30 seconds of propagation time, which translates to a total runtime of about 41 hours. This is the time I wish to reduce.
This is because future simulations will require much smaller step size, expectedly 1/20th of the above mentioned, making the total runtime in the order of weeks.

The link to the above mentioned example is - https://github.com/KapishAggarwal/tudatBundle/blob/master/tudatApplications/marsEOL/MarsEOL/AEOGraveyard_v2.cpp (I have invited you as a collaborator to this private repository.)
